src/objective-c/tests/run_tests.sh

```diff
@@ -32,10 +32,22 @@ set -e
 
 cd $(dirname $0)
 
+# Compile the C++ interop server if it doesn't exist yet. This has to be done
+# before pod install because the latter renames some C gRPC files and not the
+# interop server references to them.
+cd ../../..
+[ -f bins/dbg/interop_server ] || make CONFIG=dbg interop_server
```

Comment:
We shouldn't need to do this here. `run_tests.py` has a facility to have a make target be run before a test.